README.md

```diff
@@ -36,11 +43,15 @@ The `spec.go` file provides a programmatic representation of the configuration o
 server to validate configuration and dynamically display configuration options to end users.
 
 ### Configuration
-
+[comment]: <> (fix table format and spacing)
 | name |  description | required | default value |
-|------|---------|-------------|----------|
+|------|--------------|----------|---------------|
 |`serviceAccount`|Path to service account file with access to project|true| false|
 |`projectID`| The Project ID on endpoint|true| false|
 |`datasetID`|The dataset ID to pull data from.|true|false|
 |`tableID`|Specify comma separated table IDs. Will pull whole dataset if no Table ID present. |false|false|
 |`datasetLocation`|Specify location were dataset exist|true|false|
+|`pollingTime`|Specify number of minutes after which polling of data should be done. For eg, 60 to pull after each hour. 1 to pull every minute|false|true|
```

Comment:
I'm wondering if this is too coarse, and if we should let a user specify seconds or milliseconds.

Reply:
Changed to duration by referring S3 connector.